### PR TITLE
Bubble Sort Algorithm

### PR DESCRIPTION
**Pull request by TheShivamkumarsingh**
I have completed the algorithem please verify and merge it

### CODE REVIEW

File: BubbleSort.class
Comment (PradyumnaKrishna) [inline BubbleSort.class:1]: You don't have to include compiled class. Commit only source code.
Please delete this file and commit.

Comment (TheShivamkumarsingh) [reply BubbleSort.class:1]: Done

**Review by PradyumnaKrishna — APPROVED** [review]
LGTM! Congratulations @TheShivamkumarsingh for your first PR merge.

**Comment by TheShivamkumarsingh** [comment]
Thank you

**Pull request merged**